_data/versions.yml

```diff
@@ -1,4 +1,62 @@
 v3.1:
+- title: v3.1.3
+  components:
+    calico/node:
+      version: v3.1.3
+      url: https://github.com/projectcalico/calico/releases/tag/v3.1.3
+    calicoctl:
+      version: v3.1.3
+      url: https://github.com/projectcalico/calicoctl/releases/tag/v3.1.3
+      download_url: https://github.com/projectcalico/calicoctl/releases/download/v3.1.3/calicoctl
+    calico/cni:
+      version: v3.1.3
+      url: https://github.com/projectcalico/cni-plugin/releases/tag/v3.1.3
+      download_calico_url: https://github.com/projectcalico/cni-plugin/releases/download/v3.1.3/calico
+      download_calico_ipam_url: https://github.com/projectcalico/cni-plugin/releases/download/v3.1.3/calico-ipam
+    calico/kube-controllers:
+      version: v3.1.3
+      url: https://github.com/projectcalico/k8s-policy/releases/tag/v3.1.3
+    networking-calico:
+      version: 3.1.3
+      url: http://git.openstack.org/cgit/openstack/networking-calico/commit/?h=3.1.3
```

Comment:
@caseydavenport how is this to being available?

Reply:
Now :)